tensorflow/core/common_runtime/device.h

```diff
@@ -178,6 +178,8 @@ class Device : public DeviceBase {
 
   virtual bool IsLocal() const { return true; }
 
+  virtual bool IsRemoteCallAllowed() const = 0;
```

Comment:
Can you add a comment to this method?

Reply:
Hi qqfish
I've added brief information. Please let me know if you want me to be more elaborate.